### PR TITLE
Document changes for authorization

### PR DESCRIPTION
**Pull request by Sahiba-Gupta**
Description
This PR will update the document to pick the latest changes in authorization.

| GitHub Issue # |
](https://github.com/dell/csm/issues/368)
Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by sharmilarama — DISMISSED** [review]

**Review by rsedlock1958 — DISMISSED** [review]
I have reviewed this. NO edits are required

File: content/docs/authorization/deployment/helm/_index.md [hunk 1]
```diff
@@ -257,7 +257,7 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
    | intendedEndpoint | HTTPS REST API endpoint of the backend storage array. | Yes | - |
    | endpoint | HTTPS localhost endpoint that the authorization sidecar will listen on. | Yes | https://localhost:9400 |
    | systemID | System ID of the backend storage array. | Yes | " " |
-   | insecure | A boolean that enables/disables certificate validation of the backend storage array. This parameter is not used. | No | true |
+   | skip_certificate_validation | A boolean that enables/disables certificate validation of the backend storage array. This parameter is not used. | No | true |
```
Comment (shaynafinocchiaro) [inline content/docs/authorization/deployment/helm/_index.md:260]: This should be `skipCertificateValidation ` to stay consistent with how it is written on other doc pages

File: content/docs/authorization/deployment/rpm/_index.md [hunk 2]
```diff
@@ -230,7 +230,7 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
    | intendedEndpoint | HTTPS REST API endpoint of the backend storage array. | Yes | - |
    | endpoint | HTTPS localhost endpoint that the authorization sidecar will listen on. | Yes | https://localhost:9400 |
    | systemID | System ID of the backend storage array. | Yes | " " |
-   | insecure | A boolean that enables/disables certificate validation of the backend storage array. This parameter is not used. | No | true |
+   | skip_certificate_validation | A boolean that enables/disables certificate validation of the backend storage array. This parameter is not used. | No | true |
```
Comment (shaynafinocchiaro) [inline content/docs/authorization/deployment/rpm/_index.md:233]: Same as above

**Review by shaynafinocchiaro — CHANGES_REQUESTED** [review]

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No edits are required.

**Pull request merged**